source/reporting/index.html.md.erb

```diff
@@ -58,48 +58,154 @@ You can use the GOV.UK Pay API to:
 
 `GET /v1/payments/{paymentId}`
 
-See the <a href="https://govukpay-api-browser.cloudapps.digital/#getpayment" target="blank">GOV.UK Pay API browser</a> for more details
+Example response:
 
-If the payment exists, the JSON response body to this
-API call contains a `state` field. You can use this information when
-designing a service.
+```json
+{
+  "created_date": "2019-07-11T10:36:26.988Z",
+  "amount": 3750,
+  "state": {
+    "status": "success",
+    "finished": true,
+  },
+  "description": "Pay your council tax",
+  "reference": "12345",
+  "language": "en",
+  "metadata": { 
+    "ledger_code": "AB100",
+    "an_internal_reference_number": 200
+  },
+  "email": "sherlock.holmes@example.com",
+  "card_details": {
+    "card_brand": "Visa",
+    "card_type": "debit",
+    "last_digits_card_number": "1234",
+    "first_digits_card_number": "123456",
+    "expiry_date": "04/24",
+    "cardholder_name": "Sherlock Holmes",
+    "billing_address": {
+        "line1": "221 Baker Street",
+        "line2": "Flat b",
+        "postcode": "NW1 6XE",
+        "city": "London",
+        "country": "GB"
+    }
+  },  
+  "payment_id": "hu20sqlact5260q2nanm0q8u93",
+  "refund_summary": {
+    "status": "available",
+    "amount_available": 3750,
```

Comment:
The refund `amount_available` includes the corporate card fees, so should be 4000